scipy/sparse/linalg/_eigen/arpack/arnaud/include/arnaud/types.h

```diff
@@ -0,0 +1,23 @@
+#ifndef ARNAUD_TYPES_H
+#define ARNAUD_TYPES_H
+
+#include <stdint.h>
+#include <complex.h>
+
+
+#if defined(_MSC_VER)
+    // MSVC definition
```

Comment:
nit: if you're changing code style everywhere to `/* */`, may as well be consistent in this file as well.